# HG changeset patch
# User vinnie
# Date 1296813157 0
#      Fri Feb 04 09:52:37 2011 +0000
# Node ID 78fe1b7a9a1a51c0517e4be49c980ef6a2594f30
# Parent  fed61c2f4d14de7c88f044681331df79cb6be4da
7017176, PR2479: Several JNDI tests are mssing GPL header
Reviewed-by: alanb

diff -r fed61c2f4d14 -r 78fe1b7a9a1a test/com/sun/jndi/ldap/LdapName/Case.java
--- openjdk/jdk/test/com/sun/jndi/ldap/LdapName/Case.java	Fri Feb 04 00:33:58 2011 +0000
+++ openjdk/jdk/test/com/sun/jndi/ldap/LdapName/Case.java	Fri Feb 04 09:52:37 2011 +0000
@@ -1,3 +1,26 @@
+/*
+ * Copyright (c) 2011, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
 /*
  * @test
  * @bug 4278094
diff -r fed61c2f4d14 -r 78fe1b7a9a1a test/com/sun/jndi/ldap/LdapName/EmptyNameSearch.java
--- openjdk/jdk/test/com/sun/jndi/ldap/LdapName/EmptyNameSearch.java	Fri Feb 04 00:33:58 2011 +0000
+++ openjdk/jdk/test/com/sun/jndi/ldap/LdapName/EmptyNameSearch.java	Fri Feb 04 09:52:37 2011 +0000
@@ -1,3 +1,26 @@
+/*
+ * Copyright (c) 2011, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
 /**
  * @test
  * @bug 6997561
diff -r fed61c2f4d14 -r 78fe1b7a9a1a test/com/sun/jndi/ldap/LdapName/UnescapeTest.java
--- openjdk/jdk/test/com/sun/jndi/ldap/LdapName/UnescapeTest.java	Fri Feb 04 00:33:58 2011 +0000
+++ openjdk/jdk/test/com/sun/jndi/ldap/LdapName/UnescapeTest.java	Fri Feb 04 09:52:37 2011 +0000
@@ -1,3 +1,26 @@
+/*
+ * Copyright (c) 2011, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
 /*
  * @test
  * @bug 4892070
diff -r fed61c2f4d14 -r 78fe1b7a9a1a test/com/sun/jndi/ldap/ReadTimeoutTest.java
--- openjdk/jdk/test/com/sun/jndi/ldap/ReadTimeoutTest.java	Fri Feb 04 00:33:58 2011 +0000
+++ openjdk/jdk/test/com/sun/jndi/ldap/ReadTimeoutTest.java	Fri Feb 04 09:52:37 2011 +0000
@@ -1,3 +1,26 @@
+/*
+ * Copyright (c) 2011, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
 /**
  * @test
  * @bug 6176036
